### PR TITLE
feat(db): add a not-null constraint to name and drop title 

### PR DESCRIPTION
**Pull request by SishaarRao**
**Why: ** This is the last step for migration

**What: ** Add a not null constraint to name, and drop title

Details:
- Finally drop the title column, finishing the migration of the column rename
- Update tests to reflect the new column name
- Update seed file to reflect new column name

### CODE REVIEW

**Comment by Lianathanoj** [comment]
talked IRL about double writing the `name` column and backfilling before eventually wanting to drop the `title` column in this PR if we want to imitate a true zero downtime process; will opt to review this PR again when a requisite PR has been put up.

on another note, you'll also want to make sure to change the seeds to only use `name` here

**Review by Lianathanoj — APPROVED** [review]
after merging in your double write PR, running the backfill, and rebasing this PR, you'll still need to make changes in the controller to only write to the `name` column. furthermore, because the code changes from this PR propagate immediately, you should be good to go to run this migration to drop the `title` column to ensure zero downtime. LGTM pending the controller change and doing all the aforementioned steps!

**Review by jekatlob — APPROVED** [review]

**Pull request merged**
